release notes/v0.4.0.md

```diff
@@ -0,0 +1,169 @@
+- `locator.type(text, [options])` ([#399](https://github.com/grafana/xk6-browser/pull/399))
+- `locator.hover([options])` ([#400](https://github.com/grafana/xk6-browser/pull/400))
+- `locator.tap([options])` ([#401](https://github.com/grafana/xk6-browser/pull/401))
+- `locator.dispatchEvent(type, eventInit, [options])`([#405](https://github.com/grafana/xk6-browser/pull/405))
+- `locator.waitFor([options])` ([#406](https://github.com/grafana/xk6-browser/pull/406))
+
+
+## Bugs fixed
+
+- Fixed zombie browser processes. ([#306](https://github.com/grafana/xk6-browser/pull/306))
+
+  Before this fix, browser processes remained active even after the test or k6 test runs. This could sometimes saturate the machine and therefore could affect the tests.
+
+- Fixed cleanup of the browser's temporary directory. ([#323](https://github.com/grafana/xk6-browser/pull/323), [#409](https://github.com/grafana/xk6-browser/pull/409))
+
+  The temporary directory is now properly removed when `Browser.close()` is called, or when the iteration or test run completes. This will help ensure that any private data that the browser caches in the temporary directory is deleted.
+
+  NOTE: There is a small chance that the temporary directory will not be cleaned and a ticket has been opened to track the next step/fix.  ([#403](https://github.com/grafana/xk6-browser/issues/403))
+
+- Fixed keyboard press of modifier keys (e.g. `Shift`) in `keyboard.down` and `keyboard.up`. ([#325](https://github.com/grafana/xk6-browser/pull/325))
+
+- Fixed `page.content()` so that it no longer panics. ([#327](https://github.com/grafana/xk6-browser/pull/327))
+
+- Fixed `ElementHandle.fill()`. ([#383](https://github.com/grafana/xk6-browser/pull/383))
+
+  Before, it was not filling in the text in the selected input element.
+
+- Fixed `ElementHandle.tap()`. ([#401](https://github.com/grafana/xk6-browser/pull/401))
+
+  Before, it was performing the tap without respecting the vertical coordinate.
+
+- Fixed timeout detection. ([#411](https://github.com/grafana/xk6-browser/pull/411))
+
+  Before this fix, we couldn't detect the timeouts properly. For example if we write a test where we are retrieving the contents of an element that doesn't exist, it used to return an empty string, but now it will raise a timeout error correctly. The current system is still unstable because this fix re-surfaced a data race bug that happens from time to time (happens on frame navigation). Previously reported in [#254](https://github.com/grafana/xk6-browser/issues/254)
+
+
+## Improvements
+
+- Permissions can now be granted (such as microphone and camera) to the browser.  ([#314](https://github.com/grafana/xk6-browser/pull/314))
+
+  This grants the new browser window permissions that would then allow tests to use those devices. You can test the change using the example in [examples/grant_permission.js](https://github.com/grafana/xk6-browser/blob/v0.4.0/examples/grant_permission.js)
```

Comment:
```suggestion
  This grants the new browser window permissions that would allow tests to use those devices. You can test the change using the example in [examples/grant_permission.js](https://github.com/grafana/xk6-browser/blob/v0.4.0/examples/grant_permission.js)
```